pkg/util/log/e2e/e2e.go

```diff
@@ -0,0 +1,30 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/metrics"
+	"github.com/Azure/ARO-RP/pkg/metrics/statsd/e2e"
+)
+
+type E2EHook struct {
+	logToMetrics e2e.E2ELogToMetrics
+}
+
+func NewE2EHook(m metrics.Emitter) E2EHook {
+	return E2EHook{
+		logToMetrics: e2e.NewE2ELogToMetrics(m),
```

Comment:
use a package alias here, it's really confusing because we are in the e2e package here too

Reply:
Now using package alias `e2estatsd`.